dashboard-for-desktop/articles/dashboard-designer/appearance-customization/conditional-formatting.md

```diff
@@ -32,63 +21,80 @@ Comparison rules used in conditional formatting can be divided into the followin
 * [Bar Color Ranges](conditional-formatting/bar-color-ranges.md) - Allows you to visualize numeric values using bars whose colors are contained in the specified color set.
 * [Bar Gradient Ranges](conditional-formatting/bar-gradient-ranges.md) - Allows you to visualize numeric values using bars whose colors are contained in the specified color gradient.
 
-You can create comparison rules for [measures or dimensions](../binding-dashboard-items-to-data/binding-dashboard-items-to-data.md). The list below shows format conditions that can be applied to different types of data items.
-* Measure/numeric Dimension
+The list below shows format conditions that can be applied to different types of data items.
```

Comment:
```suggestion
Format conditions that can be applied to different types of data items are as follows.
```

Reply:
```suggestion
Format conditions that can be applied to different data item types are as follows:
```

Reply:
Is there any difference between my suggestion and your suggestion?